README.md

```diff
@@ -240,19 +241,25 @@ See section below for more information.
-| `reporters.graphite.prefix` | None               | The graphite metric prefix (if not set, prefix will be empty)                                                                         |
-| `poll-interval`             | `30 seconds`       | How often to poll Kafka for latest and group offsets                                                                                  |
-| `lookup-table-size`         | `60`               | The maximum window size of the look up table **per partition**                                                                        |
-| `client-group-id`           | `kafkalagexporter` | Consumer group id of kafka-lag-exporter's client connections                                                                          |
-| `kafka-client-timeout`      | `10 seconds`       | Connection timeout when making API calls to Kafka                                                                                     |
-| `clusters`                  | `[]`               | A statically defined list of Kafka connection details.  This list is optional if you choose to use the Strimzi auto-discovery feature |
-| `watchers`                  | `{}`               | Settings for Kafka cluster "watchers" used for auto-discovery.                                                                        |
-| `metric-whitelist`          | `[".*"]`           | Regex of metrics to be exposed via Prometheus endpoint. Eg. `[".*_max_lag.*", "kafka_partition_latest_offset"]`                       |
+| Key                           | Default              | Description                                                                                                                           |
+|-------------------------------|----------------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| `reporters.prometheus.port`   | `8000`               | The port to run the Prometheus endpoint on                                                                                            |
+| `reporters.graphite.host`     | None                 | The graphite host to send metrics to (if not set, will not output to graphite)                                                        |
+| `reporters.graphite.port`     | None                 | The graphite port to send metrics to (if not set, will not output to graphite)                                                        |
+| `reporters.graphite.prefix`   | None                 | The graphite metric prefix (if not set, prefix will be empty)                                                                         |
+| `reporters.influxDB.endpoint` | None                 | The influxDB host to send metrics to (if not set, will not output to influxDB)                                                        |
```

Comment:
Can you make the config key for `reports.influxDB` lowercase?  Lightbend config is case sensitive so this might trip some people up.

`reports.influxdb`

Reply:
You can leave the correct casing for the rest of the comments & docs.

Reply:
Sure, this is also updated as requested. Please check.